homeassistant/components/dlink/switch.py

```diff
@@ -145,14 +146,14 @@ def update(self):
                 _LOGGER.warning("Waiting %s s to retry", retry_seconds)
                 return
 
-        _state = "unknown"
+        _state = STATE_UNKNOWN
```

Comment:
This is not a home assistant state. It's the dlink api state.

Reply:
maybe `DLINK_STATE_UNKNOWN`